### ISSUE TITLE
README not in source zip

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximemaher/pyodbc#22]
```
The readme was renamed to .rst for github support, but the manifest was not
updated to include this new name.

```

Original issue reported on code.google.com by `mkleehammer` on 8 Jan 2009 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed in e673bffa17f4c782f6122fb553b89275a35b96fa

Will be in 2.15
```

Original comment by `mkleehammer` on 18 Mar 2009 at 3:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mkleehammer` on 21 Nov 2010 at 4:44
- Changed state: **Complete**
